### PR TITLE
Upstream merge 2020-02-18

### PR DESCRIPTION
**Pull request by dcbw**
@openshift/networking @danwinship

### CODE REVIEW

**Comment by dcbw** [comment]
Registry returns 500 when pushing the ovnkube image.

/retest

**Comment by danwinship** [comment]
We're still getting upstream CI. Didn't we agree to delete/revert that in the merge?

**Comment by dcbw** [comment]
> We're still getting upstream CI. Didn't we agree to delete/revert that in the merge?

@danwinship not that I recall but I may have been on PTO at that time. But we can kill it since the openshift e2e tests should be more comprehensive.

File: dist/images/ovnkube.sh [hunk 1]
```diff
@@ -551,7 +573,7 @@ metadata:
   namespace: ${ovn_kubernetes_namespace}
 subsets:
   - addresses:
-      - ip: ${ovn_db_host}
+`for ip in ${ips[@]}; do printf "    - ip: ${ip}\n"; done`
```
Comment (pecameron) [inline dist/images/ovnkube.sh]: Is the indent correct on this?

**Comment by dcbw** [comment]
```
[Conformance][Area:Networking][Feature:Router] The HAProxy router [Top Level] [Conformance][Area:Networking][Feature:Router] The HAProxy router should set Forwarded headers appropriately [Suite:openshift/conformance/parallel/minimal]
```

/test e2e-gcp-ovn

**Comment by dcbw** [comment]
`level=fatal msg="failed to fetch Terraform Variables: failed to fetch dependency of \"Terraform Variables\": failed to fetch dependency of \"Bootstrap Ignition Config\": failed to fetch dependency of \"Common Manifests\": failed to generate asset \"DNS Config\": getting public zone for \"origin-ci-int-aws.dev.rhcloud.com\": listing hosted zones: Throttling: Rate exceeded\n\tstatus code: 400, request id: 4d0d3929-15d4-4cc6-a508-4c2d70c649f1"`
/test e2e-aws-ovn

File: dist/images/ovnkube.sh (hunk 1 above)
Comment (dcbw) [reply dist/images/ovnkube.sh]: yeah I think that's tempate code? but we don't use it in openshift anyway; it's an upstream thing and should get fixed there if it's a problem.

**Comment by pecameron** [comment]
@dcbw There are OOMKilled problems that the node team is working on. See slack. Its not likely anything will pass until they are done.

**Pull request merged**